src/chapters/vstools/studio.rst

```diff
@@ -0,0 +1,22 @@
+If It Ain't Broke, Add a Breakpoint!
+====================================
+
+On your machine, open up your fork of the ``HelloASPDotNETDemo`` `repository <https://github.com/LaunchCodeEducation/HelloASPDotNETDemo>`_ in Visual Studio.
```

Comment:
can you clarify, why cant they do this studio with their own app and not the demo version?

Reply:
I got all messed up! This is a mistake! I was supposed to put the demo code in a note that specified in students were struggling with getting their own version of the app to work, they could use the demo code for the purposes of the first part of the studio.